src/serverModuleGen.py

```diff
@@ -22,6 +25,21 @@ def generateServerIncludes(config_filename, reg_filename):
 
     serverfile.write("}\n")
 
+def generateSearchPaths(filename):
+    with open(filename) as configfile:
+        modules = configfile.readlines()
+        ret = ""
+        for module in modules:
+            module = (module.rstrip()).lstrip()
+            # Represents a path to a file
+            if len(module) > 0 and module[0] == '/':
```

Comment:
You can simplify this as `if module.startswith("/")`

Reply:
Thanks for the suggestions, both are better for sure!